tests/test_filters_minicircle.py

```diff
@@ -30,21 +31,21 @@ def test_remove_quadratic_unmasked(minicircle_initial_quadratic_removal: Filters
     )
 
 
-def test_get_threshold_otsu(minicircle_threshold_otsu: np.array) -> None:
+def test_get_threshold_otsu(minicircle_threshold_otsu: Filters) -> None:
     """Test calculation of threshold."""
     assert isinstance(minicircle_threshold_otsu.thresholds, dict)
     assert minicircle_threshold_otsu.thresholds["above"] == pytest.approx(expected=[0.40108209618256363])
 
 
-def test_get_threshold_stddev(minicircle_threshold_stddev: np.array) -> None:
+def test_get_threshold_stddev(minicircle_threshold_stddev: Filters) -> None:
     """Test calculation of threshold."""
     assert isinstance(minicircle_threshold_stddev.thresholds, dict)
-    assert minicircle_threshold_stddev.thresholds == pytest.approx(
-        {"below": [-7.484708050736529], "above": [0.4990958836019106]}
+    assert dict_almost_equal(
```

Comment:
This is the sort of situation where I think [syrupy](https://github.com/syrupy-project/syrupy) can be really useful.

```suggestion
    assert minicircle_threshold_stddev.thresholds == snapshot
```